front/tests/pages/stdcm-page-model.ts

```diff
@@ -307,68 +320,134 @@ class STDCMPage {
 
   // Fills and verifies origin details with suggestions
```

Comment:
```suggestion
  // Fill and verify origin details with suggestions
```
this is a nit suggestion for this comment and every others following the same conjugation